scripts/build.js

```diff
@@ -1,6 +1,6 @@
 const esbuild = require("esbuild");
 
-const entryPoints = ["src/index.ts"];
+const entryPoints = ["src/wt.ts"];
```

Comment:
renaming this entrypoint as the generated types need to match the imported files (e.g. `wt.ts` => `dist/wt.d.ts`) which wasn't happening before